aptos-move/e2e-move-tests/src/tests/aggregator_v2.data/pack/sources/aggregator_v2_test.move

```diff
@@ -152,81 +151,71 @@ module 0x1::aggregator_v2_test {
         f(value)
     }
 
-    public entry fun new<Element: drop + copy + store>(account: &signer, use_type: u32, i: u64, limit: Element) acquires AggregatorInResource, AggregatorInTable, AggregatorInResourceGroup {
-        let addr = signer::address_of(account);
+    public entry fun new<Element: drop + copy + store>(_account: &signer, addr: address, use_type: u32, i: u64, limit: Element) acquires AggregatorInResource, AggregatorInTable, AggregatorInResourceGroup {
         insert<Aggregator<Element>>(addr, use_type, i, aggregator_v2::create_aggregator(limit));
```

Comment:
What is `use_type`?

Reply:
whether to use AggregatorInResource, AggregatorInTable or AggregatorInResourceGroup